nodes/src/nodes/chart_chartjs/IInstance.py

```
@@ -24,55 +24,134 @@
+        if chart_type:
+            q.addContext(f'Chart type: {chart_type}')
+        if title:
+            q.addContext(f'Title: {title}')
+        if description:
+            q.addContext(f'Description: {description}')
+
+        q.addGoal(
+            'Generate a Chart.js v4 configuration for the provided data' + (f' as a {chart_type} chart' if chart_type else '') + '.',
+        )
+        q.addQuestion(
+            'Generate the Chart.js configuration JSON for the data above.' if not description else description,
```

Ruff (0.15.7)

[warning] Use `description if description else 'Generate the Chart.js configuration JSON for the data above.'` instead of `'Generate the Chart.js configuration JSON for the data above.' if not description else description`

Replace with `description if description else 'Generate the Chart.js configuration JSON for the data above.'`

(SIM212)